modules/acquia_cms_image/acquia_cms_image.install

```diff
@@ -78,3 +79,20 @@ function acquia_cms_image_update_8001() {
     }
   }
 }
+
+/**
+ * Update the Image styles on Manage display pages of Image media.
+ */
+function acquia_cms_image_update_8002() {
+  $module_path = \Drupal::service('extension.path.resolver')->getPath('module', 'acquia_cms_image');
+  $config_path = $module_path . '/config/optional';
+  $source = new FileStorage($config_path);
+  $config_storage = \Drupal::service('config.storage');
+  $config_storage->write('core.entity_view_display.media.image.embedded', $source->read('core.entity_view_display.media.image.embedded'));
```

Comment:
@panshulK We can't import the whole config during update hook as it cause unforeseen issues like change the language code, overwrite any other changes made by other developers etc. ( see this: https://backlog.acquia.com/browse/ACMS-967)
We've to load the config entities and programatically we need to update the changes.
cc: @chandan-singh7929

Reply:
Done